omod/src/main/resources/config.xml

```diff
@@ -22,9 +26,13 @@
 		</init-param>
 	</filter>
 
+    <filter-mapping>
+        <filter-name>oauth2ServiceAccountFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
 	<filter-mapping>
 		<filter-name>oauth2-request-filter</filter-name>
 		<url-pattern>/*</url-pattern>
 	</filter-mapping>
```

Comment:
@wluyima can we settle on `camelCase` or `kebab-case` for the filter names?
```xml
<filter-name>oauth2-service-account-filter</filter-name>
```
?

Reply:
ok